<launch>
    

  <node pkg="rosserial_python" type="serial_node.py" name="turret_node">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baud" value="57600"/>
    <param name="angle_min" value="-3.14159265359"/>
    <param name="angle_max" value="3.14159265359"/>
    <param name="num_pts" value="32"/>
    <param name="time_increment" value="20000"/>
  </node>

<!--
  <node pkg="rosserial_python" type="serial_node.py" name="mega_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
    <param name="/sweeper/mouse_gains" value="[1,1]" />
    <param name="/sweeper/enc_gains" value="[1,1]" />
  </node>
-->
    
  <node pkg="sweeper_interface" type = "rosie_interpreter" name="rosie_interpreter" output="screen">
    <param name="angle_min" value="-3.14159265359"/>
    <param name="angle_max" value="3.14159265359"/>
    <param name="num_pts" value="32"/>
    <param name="time_increment" value="0.02"/>
  </node>


</launch>
